app/domain/authentication/authn_k8s/inject_client_cert.rb

```diff
@@ -71,7 +72,7 @@ def validate
       def install_signed_cert
         pod_namespace = spiffe_id.namespace
         pod_name = spiffe_id.name
-        cert_file_path = "/etc/conjur/ssl/client.pem"
+        cert_file_path = @cert_path.nil? || @cert_path.empty? ? DEFAULT_CERT_PATH : @cert_installation_path
```

Comment:
i may be missing something but where is `@cert_path` defined?

Reply:
oh i guess it should be `@cert_installation_path`, correct?

Reply:
👍